src/main/java/org/gogame/server/domain/entities/dto/GameMessageDto.java

```diff
@@ -0,0 +1,23 @@
+package org.gogame.server.domain.entities.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class GameMessageDto {
+
+    private Long author;
+
+    private Long userSenderId;
+
+    private Long userReceiverId;
```

Comment:
get rid of both `userSenderId` and `userReceiverId`, add a `gameId` and change `author` to `authorId`

Reply:
I rewrote it from your dto specifications. I will change it in the next PR